docs/guides/monitor/kubernetes-k8s-netdata.md

```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
 Without this visibility, it's like you built an entire house and _then_ smashed your way through the finished walls to
 add windows.
 
-At Netdata, we're working to build Kubernetes monitoring tools that add visibility without complexity while also helping
-you actively troubleshoot anomalies or outages. Better yet, this toolkit includes a few complementary collectors that
-let you monitor the many layers of a Kubernetes cluster entirely for free.
+## Solution
 
-We already have a few complementary tools and collectors for monitoring the many layers of a Kubernetes cluster,
-_entirely for free_. These methods work together to help you troubleshoot performance or availability issues across
-your k8s infrastructure.
-
--   A [Helm chart](https://github.com/netdata/helmchart), which bootstraps a Netdata Agent pod on every node in your
-    cluster, plus an additional parent pod for storing metrics and managing alarm notifications.
--   A [service discovery plugin](https://github.com/netdata/agent-service-discovery), which discovers and creates
-    configuration files for [compatible
-    applications](https://github.com/netdata/helmchart#service-discovery-and-supported-services) and any endpoints
-    covered by our [generic Prometheus
-    collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/prometheus). With these
-    configuration files, Netdata collects metrics from any compatible applications as they run _inside_ of a pod.
-    Service discovery happens without manual intervention as pods are created, destroyed, or moved between nodes. 
--   A [Kubelet collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet), which runs
-    on each node in a k8s cluster to monitor the number of pods/containers, the volume of operations on each container,
-    and more.
--   A [kube-proxy collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy), which
-    also runs on each node and monitors latency and the volume of HTTP requests to the proxy.
--   A [cgroups collector](/collectors/cgroups.plugin/README.md), which collects CPU, memory, and bandwidth metrics for
-    each container running on your k8s cluster.
-
-By following this guide, you'll learn how to discover, explore, and take away insights from each of these layers in your
-Kubernetes cluster. Let's get started.
-
-## Prerequisites
-
-To follow this guide, you need:
-
--   A working cluster running Kubernetes v1.9 or newer.
--   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool, within [one minor version
+In this tutorial, you'll learn how to navigate Netdata's Kubernetes monitoring features, using
+[robot-shop](https://github.com/instana/robot-shop) as an example deployment. Deploying robot-shop is purely optional.
```

Comment:
It's awesome that we are using a real-life example. Let's try to put into our workflow the ideation of project examples that have a co-marketing/co-community value as well (e.g K6: addition of netdata in their docs, etc). I am thinking to play with Netdata + OpenFaaS which is deployed on K8s.

Reply:
With a bit more time to validate and plan these things, that would be awesome.

Reply:
That's a very very very valid comment. I will make a point today in product-marketing sync to underline that an additional benefit to having more time is to scout for opportunities just like this one.